### PR TITLE
Aur scripts added

### PR DESCRIPTION
**Pull request by WafiWadud**
PKGBUILD for aur, need you to accept this before I can actually add this to the aur.
[for this issue](https://github.com/aartaka/pretty.c/issues/1)

### CODE REVIEW

**Comment by WafiWadud** [comment]
please add this to the aur. To do so please register into the aur and add the ssh keys into your keyring. after which it's just as simple as:
```bash
git remote add origin ssh://aur@aur.archlinux.org/pretty-header.git
git push -u origin master
```

**Comment by WafiWadud** [comment]
To register for an account on the Arch User Repository (AUR), follow these steps:

Visit the AUR website:
Go to https://aur.archlinux.org/
Click on "Register":
In the top right corner of the page, you should see a "Register" link. Click on it.
Fill out the registration form:
You'll need to provide the following information:

Username: Choose a unique username
Email: Use a valid email address
Password: Choose a strong password
Confirm password: Re-enter your password
Language: Select your preferred language
SSH Public Key: This is optional at registration, but you'll need it later to upload packages


Complete the CAPTCHA:
Solve the CAPTCHA challenge to prove you're not a bot.
Agree to the terms:
Read and agree to the terms of service.
Submit the form:
Click on the "Register" button to create your account.
Verify your email:
Check your email for a verification link. Click on this link to activate your account.
Log in:
Once your email is verified, you can log in to your new AUR account.

After registration, it's important to set up your SSH key if you didn't during registration:

Log in to your AUR account
Go to "My Account"
Scroll down to the "SSH Public Key" section
Paste your SSH public key into the text box
Click "Update" to save your key

Remember, your SSH key is crucial for package maintenance on AUR. If you don't have an SSH key yet, you'll need to generate one. Here's how you can do that:
```bash
# Generate a new SSH key
ssh-keygen -t ed25519 -C "your_email@example.com"

# Start the ssh-agent in the background
eval "$(ssh-agent -s)"

# Add your SSH private key to the ssh-agent
ssh-add ~/.ssh/id_ed25519

# Display your public key to copy to AUR
cat ~/.ssh/id_ed25519.pub
```
After generating your SSH key, copy the output of the last command (your public key) and paste it into the SSH Public Key field in your AUR account settings.
Is there any part of the registration process you'd like me to explain further?

File: src/pretty.h [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/wafi/coding/pretty.c/pretty.h
```
Comment (aartaka) [inline src/pretty.h:1]: This symlink is absolute, not relative. Can you fix that?

File: pkg/pretty-c/.BUILDINFO [hunk 2]
```diff
@@ -0,0 +1,1856 @@
...
+options = strip
+options = docs
+options = !libtool
+options = !staticlibs
+options = emptydirs
+options = zipman
+options = purge
+options = debug
+options = lto
+installed = a52dec-0.8.0-2-x86_64
```
Comment (aartaka) [inline pkg/pretty-c/.BUILDINFO]: I'm not sure all of these are needed to "build" pretty.c. Can you cull the list down?

File: .SRCINFO [hunk 3]
```diff
@@ -0,0 +1,13 @@
+pkgbase = pretty-c
+	pkgdesc = Make C look like many other languages while retaining its charecteristics. not how god intended.
```
Comment (aartaka) [inline .SRCINFO]: Maybe use the repository description or README title instead?

**Review by aartaka — COMMENTED** [review]
I can't claim to understand everything about it, but some things feel off. Can you explain the commented parts to me?

**Comment by aartaka** [comment]
@WafiWadud I'm also pretty sure that my SSH keys are not necessary to package things for AUR. And I'm not yet ready to commit to packaging/maintaining the package in AUR. So I'd prefer someone else handle that. Feel free to jump in and do packaging decisions, I approve (with these comments addressed, of course).

File: pkg/pretty-c/.BUILDINFO (hunk 2 above)
Comment (WafiWadud) [reply pkg/pretty-c/.BUILDINFO]: no, it's just like that no matter what

Comment (aartaka) [reply pkg/pretty-c/.BUILDINFO]: Okay then.

**Comment by WafiWadud** [comment]
all of the stuff i made is in the PKGBUILD file. The src info and other files are byproducts needed to upload to the aur

**Comment by WafiWadud** [comment]
uploaded to the aur

**Comment by aartaka** [comment]
Thanks!

**Pull request closed without merging**